src/Compilers/CSharp/Portable/Syntax/SyntaxFactory.cs

```diff
@@ -2513,5 +2537,61 @@ public static UsingDirectiveSyntax UsingDirective(NameEqualsSyntax alias, NameSy
                 name: name,
                 semicolonToken: Token(SyntaxKind.SemicolonToken));
         }
+
+        public static ForEachStatementSyntax ForEachStatement(TypeSyntax type, SyntaxToken identifier, ExpressionSyntax expression, StatementSyntax statement)
+        {
+            return ForEachStatement(Token(SyntaxKind.ForEachKeyword), Token(SyntaxKind.OpenParenToken), type, identifier, Token(SyntaxKind.InKeyword), expression, Token(SyntaxKind.CloseParenToken), statement);
+        }
+
+        public static ForEachStatementSyntax ForEachStatement(TypeSyntax type, string identifier, ExpressionSyntax expression, StatementSyntax statement)
+        {
+            return ForEachStatement(type, Identifier(identifier), expression, statement);
+        }
+
+        public static ForEachStatementSyntax ForEachStatement(SyntaxToken forEachKeyword, SyntaxToken openParenToken, TypeSyntax type, SyntaxToken identifier, SyntaxToken inKeyword, ExpressionSyntax expression, SyntaxToken closeParenToken, StatementSyntax statement)
+        {
+            return ForEachStatement(forEachKeyword, openParenToken, type, identifier, null, inKeyword, expression, closeParenToken, statement);
+        }
+
+        internal static VariableDeclarationSyntax VariableDeclaration()
```

Comment:
We don't need it. I forgot to remove it after making source generator changes. Thanks